python/mlx/nn/__init__.py

```diff
@@ -1,5 +1,5 @@
 # Copyright © 2023 Apple Inc.
 
-from mlx.nn import losses
+from mlx.nn import initializers, losses
```

Comment:
How do you feel about changing the name to `init`? It is much shorter / easier to type :) and PyTorch uses it (we tend to follow PyTorch nn naming conventions).